test/example.c

```diff
@@ -685,7 +685,7 @@ int main(argc, argv)
     (void)argv;
 #else
     if (argc > 1) {
-        if (strcmp(argv[1], "--appveyor") == 0) {
+        if (strcmp(argv[1], "--junit") == 0) {
```

Comment:
Nice.  But the hardcoded path is still a problem; if you run from the build directory (as out of tree builds tend to do on Linux), it fails to find the output file's directory:

```1: Test command: /home/dank/src/zlib/btmp/example "--appveyor"
1: Test timeout computed to be: 10000000
1: Could not open junit filezlib version 1.2.11.1-motley = 0x12b1, compile flags = 0xa9
```

Reply:
If you add --junit to the tests in CMakeLists.txt, both example and example64 output to the same file, and the results of one are overwritten.  Hrm.